tests/integration/targets/inventory/files/reference_fields.now.yml

```diff
@@ -11,4 +11,5 @@ compose:
   location_name: lookup('ansible.builtin.vars', 'location.name')
   street: lookup('ansible.builtin.vars', 'location.street')
 
-strict: true
+strict: false
+display_value: false
```

Comment:
Just to double-check if this change is related to this PR.

Reply:
no it's not. Sorry